sdk/storage/azure-storage-blob-changefeed/samples/change_feed_samples.py

```diff
@@ -87,16 +89,37 @@ def list_events_using_continuation_token(self):
         token = change_feed.continuation_token
 
         # restart using the continuation token
-        change_feed2 = cf_client.list_changes(results_per_page=3).by_page(continuation_token=token)
+        change_feed2 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token)
         change_feed_page2 = next(change_feed2)
         for event in change_feed_page2:
             print(event)
 
+    def list_events_in_live_mode(self):
+        # Instantiate a ChangeFeedClient
+        cf_client = ChangeFeedClient("https://{}.blob.core.windows.net".format(self.ACCOUNT_NAME),
+                                     credential=self.ACCOUNT_KEY)
+
+        start_time = datetime(2020, 9, 1, 1)
+        token = None
+
+        while True:
+            # start_time will be ignored if start_time and continuation_token are both non-empty
+            change_feed = cf_client.list_changes(start_time=start_time).by_page(continuation_token=token)
+
+            for page in change_feed:
+                for event in page:
+                    print(event)
+            token = change_feed.continuation_token
+
+            sleep(60)
+            print("continue printing events")
```

Comment:
I wanted to enable users to start listing events from a specific time. That’s why I removed the start_time and continuation toke mutual exclusive check. I feel the code is simple in this way. I also added a reminder in the doc that start time will be ignored if it’s set together with continuation token.

I can add revert the change if you strongly feel the doc is not as safe as throw exception.

Reply:
Think about what is least possible to surprise customer and how much time does it take they find out they have a bug.
I would say those are options in general (sorted by how fast feedback is):
- compiler enforced contract gives fastest feedback (which in python is grey area I know). You can't run program until you fix code.
- validation (as soon as they run program it blows up, please take a look at first page here https://www.martinfowler.com/ieeeSoftware/failFast.pdf )
- documentation. this can be ignored initially or misunderstood by user. they'll most probably pay closer attention after long debugging session.

Reply:
Yeah I can get you, I can revert this one since the point is just to let the customer listing events with a start time continuously using one while loop. Users can use a workaround by getting the continuation token first then start the while loop.

How about the end_time one? If we make end_time and continuation token mutual exlusive, those user only with a continuation token cannot list range of events.

Reply:
same here, let's not mix them.
Let's use (start time, end time) tuple to define iteration boundary and continuation just to progress through. If user wants to change boundaries they should start new CF.

Reply:
Okay! I will make start_time/end_time and continuation token mutual exclusive.
Also I will create a github issue in case users want to specify end_time for a continuation token without an end_time. Does it sound good?

Reply:
I don't suspect people running into such problem if we make API explicit (validation). I'd suggest to hold with creating such issue as this would potentially confuse people who are exploring feature in general (I think that attempt to override token's end date isn't valid use case).

Reply:
I feel It's not to override.
If someone just started listing events, and he paused the programe, then he got a token. After several days he wants to continue listing event, while he wants to have an end_date....

Reply:
We have existing example in code base to let the user pass in another value to give user more options.
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py#L200-L205

Reply:
I think more than likely a user is going to use the continuation token API when they are just trying to infinitely loop over events (so endTime is null or Max). For now at least I can't see a situation where someone wants to stop listing earlier - and even then the customer can do this by filtering for events and stopping the iteration when they hit a certain date

Reply:
Yeah, I can get you. so I'm asking if we want to create an issue in case users want that, currently python hasn't received any users feedback of changefeed, we don't know what they want.

Reply:
As for making an issue, I think we should let the request come from the customer if they really do want it. I wonder if creating a github issue might invite the idea into their head which isnt ideal

Reply:
I would rather steer customer away from using api that way. Also [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

Reply:
Okay to not steer the customer away, I made them mutual exclusive and I will not create an issue 😀